include/cocaine/repository.hpp

```diff
@@ -89,7 +89,7 @@ class repository_t {
    ~repository_t();
 
     void
-    load(const std::string& path);
+    load(const std::vector<std::string>& plugin_dirs);
```

Comment:
it's a better naming, because "paths" can be confusing (one can think it's the final paths to plugin, not to dirs containing them)